src/Containers/CollectionMultiple.php

```diff
@@ -57,7 +57,7 @@ public function getTemplate($index = '::n::')
      */
     public function pushVal($value = null)
     {
-        if (!isset($value[$this->keyField])) {
+        if (empty($value[$this->keyField])) {
```

Comment:
The point here is to know whether or not this value is defined, not if it's empty or not. The values "0" or even "" are perfectly valid.